packages/configs/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@penumbra-zone/configs",
+  "version": "1.0.0",
+  "license": "MIT",
+  "description": "Shared Penumbra configurations",
+  "repository": "penumbra-zone/web",
+  "type": "module",
+  "scripts": {},
```

Comment:
our current web <> prax local dev workflow is

**web:** `pnpm install && pnpm build && pnpm dev:pack`
**web:** `pnpm dev` 
**prax:** `pnpm -w add:tgz ~/PATH_TO_PRAX_REPO/web/packages/*/penumbra-zone-*.tgz && pnpm -w syncpack fix-mismatches && pnpm dev` 

can we modify the scripts to add build and pack steps so this package bundles a .tgz and is watched during prax / dex explorer development?

```
  "scripts": {
    "build": "tsc --build --verbose",
    "dev:pack": "tsc-watch --onSuccess \"$npm_execpath pack\"",
  },
```

correspondingly, the concurrency settings in the [package.json](https://github.com/penumbra-zone/web/blob/main/package.json#L19) will need to be updated to **17**.

Reply:
I guess this isn't _strictly_ necessary for dev since we shouldn't expect the eslint / prettier configs to change often.

Reply:
I agree that the configs are not expected to change often or dramatically - and _they shouldn't, by design_. LInts should not get in our way, but rather keep us safe from missteps.
I am testing with prax wallet now.

Reply:
are these instructions in a readme anywhere? would be helpful to have in the prax repo

Reply:
> are these instructions in a readme anywhere? would be helpful to have in the prax repo

we need to clarify these instructions in prax: https://github.com/prax-wallet/prax/issues/149